components/prize_pool/commons/prize_pool_import.lua

```diff
@@ -204,6 +206,18 @@ function Import._computeBracketPlacementEntries(matchRecords, options)
 			end)
 		end
 	)
+
+	for slotIndex, slotEntries in pairs(slots) do
```

Comment:
```suggestion

	-- Remove `bye`s from the bracket
	for slotIndex, slotEntries in pairs(slots) do
```

Reply:
at that point we already have slots and not the bracket anymore, so will use `-- Remove `bye`s from slotentries` instead